ibis/backends/dask/tests/execution/test_operations.py

```diff
@@ -168,7 +169,7 @@ def test_aggregation_group_by(t, df, where, ibis_func, dask_func):
     )
     lhs = result[expected.columns]
     rhs = expected
-    tm.assert_frame_equal(lhs.compute(), rhs.compute())
+    tm.assert_frame_equal(lhs.compute(), rhs)
```

Comment:
ideally do or similar
```
result=
expected=
tm.assert_frame_equal(result, expected)
```

Reply:
I can make this change. 

I flipped to just doing the computation at the top so I could keep the `.agg` as is (this was easier to represent in straight pandas).

Reply:
sure you can do it anywhere, just as long as the comparison is like this (the ordering doesn't matter here)